data/test/dict/ja/en/term_bank_1.json

```diff
@@ -557,35 +529,7 @@
           {
             "tag": "div",
             "content": [
-              "to lead or extend in a certain direction",
-              {
-                "tag": "div",
-                "data": {
-                  "content": "extra-info"
-                },
-                "content": {
-                  "tag": "div",
-                  "data": {
-                    "content": "example-sentence"
-                  },
-                  "content": [
-                    {
-                      "tag": "div",
-                      "data": {
-                        "content": "example-sentence-a"
-                      },
-                      "content": "山脈が南北に走る。\nSanmyaku ga nanboku ni hashiru.\nThe mountain range runs north–south."
```

Comment:
I think there are lots of examples like this one, where the translation is not parsed correctly and included in the example content, so I chose to allow examples with no translation field (and increased allowed length for them to 90). 

Besides, I think examples with no translation are still useful, especially since a user can just scan the words in the example if they don't know it. 

So what i would do is leave the data in the dict, and examples with no translation can maybe optionally be hidden with CSS like:
```css
[data-sc-content="example-sentence"]:has([data-sc-content="example-sentence-b"]:empty) {
  display: none;
}
```

Reply:
You make a good point. I can just throw that CSS into my Yomitan settings. I'll close the pull request.